CalcPi.java

```diff
@@ -1,6 +1,22 @@
 // Computes an approximation of PI.
 public class CalcPi {
 	public static void main(String [] args) { 
-	    // Replace this comment with your code
+	    int numOfTerms = Integer.parseInt(args[0]);
+		double approximated = 1;
+		int i = 1;
+		int curIndex = 1;
+		while(curIndex < numOfTerms) { 
+			curIndex ++;
+			if(curIndex % 2 == 0) {
```

Comment:
1. try shorthand ops 
2. if you have a code part that appear in all if else cases can be put outside
```
i = i + 2;
if(curIndex % 2 == 0) {
    approximated = approximated - (double) 1 / i;
} else {
    approximated = approximated + (double) 1 / i;
}
```
i